include/cell/copy/mod.hpp

```diff
@@ -1,12 +1,12 @@
 #pragma once
 
 #include "cell/copy/constants.hpp"
+// TODO(KuangjuX): Clean up `cell/copy/copy.hpp` will cause serious compilation errors, 
```

Comment:
Clean up `cell/copy/copy.hpp` will cause serious compilation errors and the reason is currently unclear.